### ISSUE TITLE
Hovertext for dropped tx state

### ISSUE THREAD

**Issue by wbt** [MetaMask/metamask-extension#6963]
**What problem are you trying to solve?**
Recently, I had a transaction enter status "dropped."  It took me a long time to figure out what that meant (as in, why did it enter that status?).  I found the answer [here](https://github.com/MetaMask/Design/issues/25#issuecomment-404677979):
 
>Dropped – the tx nonce was already used

Thanks to @bdresser for writing that comment!  Once I knew this was all it meant, debugging the cause of that became a great deal easier, and the workaround (resetting the account) also became easier.

**Describe the solution you'd like**
Hovertext on the status "Dropped" which at least contains the contents of the quote above, and possibly also ". Verify that you are using the expected account. Resetting the account in Settings->Advanced may help." so that users can figure out what to do from there.

Some of the other statuses could probably use hovertext too.

**Comment by bdresser** [comment]
@wbt hovertext for these would be great.

you saw your transaction history actually say "Dropped" ? Had you tried a cancellation or speed up?

**Comment by wbt** [comment]
@bdresser Yep, it actually said "dropped" in the transaction history, like others might say "pending" or "confirmed." It took me a while and your comment to find out what that meant, but I believe it was accurate.
